apollo-router/src/services/layers/persisted_queries/manifest_poller.rs

```diff
@@ -50,8 +50,10 @@ pub(crate) enum FreeformGraphQLBehavior {
     },
 }
 
-/// Describes what the router should do for a given request: allow it, deny it
-/// with an error, or allow it but log the operation as unknown.
+/// How we should treat requests containing freeform GraphQL rather than a PQ ID.
+/// (This also includes the results of APQ lookups, for the purpose of differenting
+/// between Allow and AllowAndLog; APQ cannot be enabled in any mode that allows
+/// freeform requests to be denied.)
 pub(crate) enum FreeformGraphQLAction {
```

Comment:
This isn't what I was suggesting. The thrust of my suggested change was that the description of each enum variant was associated directly with it. If you run `cargo doc --open` you can see the impact of the difference between documenting as is (or with this change) vs documenting as this:
```
/// How should we treat Freeform GraphQL
pub(crate) enum FreeformGraphQLAction {
    /// Allow it
    Allow,
    /// Deny it, with an error
    Deny,
    /// Allow it, but log the operation as unknown
    AllowAndLog,
    /// Deny it, with an error and log the operation as unknown <- Presumably...
    DenyAndLog,
}
```
The documentation of each variant is explicitly with the variant and it's impossible for the documentation of the enum and the variants to be misconstrued or get out of sync over time.